docs/deploying/aws/README.md

```diff
@@ -1,91 +1,33 @@
 # AWS CloudCost Exporter Deployment
 
-## Authentication
+## 1. Setup the IAM role
 
-cloudcost-exporter uses [AWS SDK for Go V2](https://docs.aws.amazon.com/sdk-for-go/v2/developer-guide/getting-started.html) and supports providing authentication via the [AWS SDK's default credential provider chain](https://docs.aws.amazon.com/sdk-for-go/v2/developer-guide/security_iam_service-with-iam.html).
+cloudcost-exporter uses [AWS SDK for Go V2](https://docs.aws.amazon.com/sdk-for-go/v2/developer-guide/getting-started.html)
+and supports providing authentication via the [AWS SDK's default credential provider chain](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html).
 This means that the CloudCost Exporter can be deployed on an EC2 instance, ECS, EKS, or any other AWS service that supports IAM roles for service accounts.
 
-You will need to create a [service role](https://docs.aws.amazon.com/sdk-for-go/v2/developer-guide/security_iam_service-with-iam.html#security_iam_service-with-iam-roles-service) and its policy.
-Below are some options for how to create one.
-
-### Option 1: AWS Console
-
-First, create a policy with the following JSON (see: [./permissions-policy.json](./permissions-policy.json)):
-```json
-{
-    "Version": "2012-10-17",
-    "Statement": [
-        {
-            "Effect": "Allow",
-            "Action": [
-                "ce:*",
-                "ec2:DescribeRegions",
-                "ec2:DescribeInstances",
-                "ec2:DescribeSpotPriceHistory",
-                "ec2:DescribeVolumes",
-                "pricing:GetProducts"
-            ],
-            "Resource": "*"
-        }
-    ]
-}
-```
-
-To create a role, for `Trusted entity type`, select `AWS service`.
-For `Use case`, select `EC2`.
-For the policy, select the one that was created in the earlier step.
-The trust policy should look like this (see: [./role-trust-policy.json](./role-trust-policy.json)):
-```json
-{
-    "Version": "2012-10-17",
-    "Statement": [
-        {
-            "Effect": "Allow",
-            "Action": [
-                "sts:AssumeRole"
-            ],
-            "Principal": {
-                "Service": [
-                    "ec2.amazonaws.com"
-                ]
-            }
-        }
-    ]
-}
-```
+First, create an IAM Policy the minimum required permissions for cloudcost-exporter to work on AWS, an example of which can be found in [./permissions-policy.json](./permissions-policy.json).
```

Comment:
Looks like `permissions-policy.json` is missing to be committed ?

Reply:
This was added in a previous PR and didn't change! https://github.com/grafana/cloudcost-exporter/blob/main/docs/deploying/aws/permissions-policy.json